tests/retryable-reads/retryable-reads-001.phpt

```diff
@@ -0,0 +1,58 @@
+--TEST--
+Retryable reads: supported single-statement operations include transaction IDs
```

Comment:
This title doesn't seem accurate. Consider "Retryable reads: find command is retried once".

EDIT: Noticed below that this was using `executeReadCommand()`. In that case, perhaps we should include the method name in the title as well if we'll have another test for `executeQuery()`.